discord/__init__.py

```diff
@@ -40,10 +40,12 @@
 from .integrations import *
 from .invite import *
 from .template import *
+from .welcome_screen import *
 from .widget import *
 from .object import *
 from .reaction import *
 from . import utils, opus, abc, ui
+from .welcome_screen import WelcomeScreen, WelcomeChannel
```

Comment:
Is this needed?
Line 34 - 37 in welcome_screen.py:
```py
__all__ = (
    'WelcomeChannel',
    'WelcomeScreen',
)
```

Reply:
CMIW, `WelcomeScreen`  and `WelcomeChannel` is imported twice in `__init__.py` (Line 43 and 48)